lua/nvim-treesitter/query.lua

```diff
@@ -120,6 +185,39 @@ function M.iter_prepared_matches(query, qnode, bufnr, start_row, end_row)
           if pred[1] == "set!" and type(pred[2]) == "string" then
             insert_to_path(prepared_match, split(pred[2]), pred[3])
           end
+          if pred[1] == "strip!" and #pred == 3 then
+            local query_name = query.captures[pred[2]]
+            local regex = pred[3]
+
+            local node = utils.get_at_path(prepared_match, query_name..'.node')
+            local ts_utils = require 'nvim-treesitter.ts_utils'
```

Comment:
Please move that above, otherwise it will be critical in terms of performances

Reply:
This is needed to avoid a circular dependency. But maybe I can avoid `get_node_text` by using `match_line`

Reply:
Circular dependency was fixed in master by style checking.